contents/docs/new-to-posthog/switch-guide/switching-to-posthog.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: "Switching to PostHog"
+nextPage: ./planning-your-pilot.mdx
+---
+
+For new projects and companies, PostHog is an easy sell: one login and one easy integration gets you everything you need to build great products.
+
+But convincing your teammates to switch horses later in a product's life can require some planning and diplomacy. We made you a cheatsheet to help make the case.
+
+## Sharing social proof
```

Comment:
I think we should move this section down. Social proof is more supporting than the main reason someone will be convinced. Maybe the right place for this is at the bottom. Definitely worth including though.